src/brisk/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ include_directories(include)
 
 #build the brisk library dynamic and static versions
 camodocal_library(camodocal_brisk SHARED ${BRISK_SOURCE_FILES} ${BRISK_HEADER_FILES})
-camodocal_link_libraries(camodocal_brisk agast ${OPENCV_CORE_LIBRARY})
+camodocal_link_libraries(camodocal_brisk agast ${OpenCV_LIBS})

```

Comment:
this expands the number of things linked dramatically compared to before.... ditto for all other cases of this